cmd/node/main.go

```diff
@@ -263,6 +263,7 @@ func setupPrivi(nodeConfig *config.NodeConfig) (*privi.Server, func()) {
 	priviServer := privi.NewServer(
 		perms,
 		repo,
+		nil,
```

Comment:
why is this nil?

Reply:
oh i guess we don't use this build anymore ... maybe we should delete this `main.go` then.